lib/builder/generator.js

```diff
@@ -141,15 +142,25 @@ module.exports = class Generator {
   }
 
   async afterGenerate() {
-    const indexPath = join(this.distPath, 'index.html')
-    if (existsSync(indexPath)) {
-      // Copy /index.html to /200.html for surge SPA
-      // https://surge.sh/help/adding-a-200-page-for-client-side-routing
-      const _200Path = join(this.distPath, '200.html')
-      if (!existsSync(_200Path)) {
-        await copy(indexPath, _200Path)
-      }
+    let { fallback } = this.options.generate
+
+    // Disable SPA fallback if value isn't true or a string
+    if (fallback !== true && typeof fallback !== 'string') return
+
+    // True means we assume the path is 404.html (for GitHub Pages, Surge, etc.)
+    if (fallback === true) fallback = '404.html'
```

Comment:
This normalize would be better moved into Options.from so that all modules will see the same option.

Reply:
Ah, totally!